src/components/OrderForm/OrderForm.js

```diff
@@ -408,14 +445,24 @@ class OrderForm extends React.Component {
           moveable={moveable}
           removeable={removeable}
           onRemove={onRemove}
-          extraIcons={(
-            !helpOpen && apiClientConnected && currentLayout && currentLayout.customHelp && (
+          extraIcons={[
+            !helpOpen && currentLayout && currentLayout.customHelp && (
               <Icon
                 name='question'
                 onClick={this.onToggleHelp}
               />
-            )
-          )}
+            ),
+            !helpOpen && currentLayout && currentLayout.id && (
+              <AOParamSettings
+                algoID={currentLayout.id}
+                symbol={activeMarket.wsID}
+                processAOData={this.processAOData}
+                setFieldData={this.setFieldData}
+                validateAOData={this.validateAOData}
+                updateValidationErrors={this.updateValidationErrors}
+              />
+            ),
+          ]}
```

Comment:
seems like it's not hidden for hosted app, so wanted to know if backend events like `algo_order_params.*` are supported with hosted backend too ? 
If not we should hide it for hosted app

Reply:
@avsek477 is currently working on adding this feature to hosted backend too

Reply:
on hosted, this feature is under code review